plugins/engines/ospray/ispc/render/utils/AbstractRenderer.ispc

```diff
@@ -347,83 +351,111 @@ inline varying vec4f
+            //            const vec4f emissionIntensity =
+            //                make_vec4f(self->emissionIntensitiesMap[normalizedValue],
+            //                0.f);
+            //            vec4f voxelColor =
+            //                max(make_vec4f(0.f), emissionIntensity +
+            //                colorMapColor);
+
+            //            // Shadow intensity
+            //            float shadowIntensity = 0.f;
+            //            vec4f giContribution = make_vec4f(0.f);
+            //            if (self->shadows > 0.f && voxelColor.w > 0.01f)
+            //            {
+            //                // Determine global illumination contribution
+            //                for (uniform int i = 0; self->lights && i <
+            //                self->numLights;
+            //                     ++i)
+            //                {
+            //                    const uniform Light* uniform light =
+            //                    self->lights[i];
+            //                    const varying vec2f s =
+            //                        make_vec2f(1.f / self->randomNumber);
+            //                    DifferentialGeometry dg;
+            //                    dg.P = point;
+            //                    const varying Light_SampleRes lightSample =
+            //                        light->sample(light, dg, s);
+
+            //                    Ray lightRay = ray;
+            //                    if (self->softShadows > 0.f)
+            //                        lightRay.dir = normalize(
+            //                            lightSample.dir +
+            //                            self->softShadows *
+            //                                getRandomVector(sample,
+            //                                lightSample.dir,
+            //                                                self->randomNumber));
+            //                    else
+            //                        lightRay.dir = lightSample.dir;
+
+            //                    lightRay.t =
+            //                    length(make_vec3f(self->volumeDimensions));
+            //                    lightRay.t0 = dg.epsilon;
+            //                    lightRay.org = point;
+            //                    lightRay.dir = lightSample.dir;
+
+            //                    shadowIntensity +=
+            //                        getVolumeShadowContribution(self,
+            //                        lightRay, sample);
+            //                }
+            //                const float giAttenuation =
+            //                    1.f - shadowIntensity * self->shadows;
+            //                voxelColor.x = voxelColor.x * giAttenuation;
+            //                voxelColor.y = voxelColor.y * giAttenuation;
+            //                voxelColor.z = voxelColor.z * giAttenuation;
+            //            }
```

Comment:
lots of commented code here, is there a good reason not to delete it?

Reply:
@favreau has to adapt the shaders to use the ospray volumes.